src/main/java/org/example/App.java

```diff
@@ -9,15 +9,19 @@ public static List<String> fizz(int number) {
         for (int i = 1; i <= number; i++) {
             boolean isDivisibleByThreeWithoutRest = i % 3 == 0;
             boolean isDivisibleByFiveWithoutRest = i % 5 == 0;
-            if (isDivisibleByThreeWithoutRest) {
+            boolean isDivisibleByBoth = i % 3 == 0 & i % 5 == 0;
+
+            if (isDivisibleByBoth) {
+                numbers.add("fizzbuzz");
+            } else if (isDivisibleByThreeWithoutRest) {
```

Comment:
```suggestion
                numbers.add("fizzbizz");
            } else if (isDivisibleByThreeWithoutRest) {
```
they wanted sonething else i think